src/sentry/search/snuba/executors.py

```diff
@@ -491,24 +494,92 @@ def has_sort_strategy(self, sort_by: str) -> bool:
         return sort_by in self.sort_strategies.keys()
 
 
+@dataclasses.dataclass
+class BetterPriorityParams:
```

Comment:
Can you move this to the top of the file near `PrioritySortWeights` and whatnot?